example/src/main/java/com/stripe/example/activity/PaymentSessionActivity.java

```diff
@@ -190,6 +190,6 @@ private ShippingInformation getExampleShippingInfo() {
                 .setPostalCode("94107")
                 .setState("CA")
                 .build();
-        return new ShippingInformation(address, "Fake Name", "6504604645");
+        return new ShippingInformation(address, "Fake Name", "(650) 460-4645");
```

Comment:
Let's update to a 555 number (650-555-4645 will suffice) so as not to anger the owner of 4604545